Dockerfile

```diff
@@ -487,6 +487,9 @@ RUN jupyter-nbextension disable nb_conda --py --sys-prefix && \
     jupyter-nbextension disable nbpresent --py --sys-prefix && \
     jupyter-serverextension disable nbpresent --py --sys-prefix
 
+# Install papermill so it can replace nbconvert as a method to execute notebooks.
+RUN pip install papermill
```

Comment:
Is it worth creating its own Docker layer for this package? Consider adding it at line 424.

If you do have to run this command at the end, make sure to call the `/tmp/clean-layer.sh` script to reduce the layer size.

Reply:
Moved thanks :)